README.md

```diff
@@ -89,29 +87,15 @@ try {
 }
 ```
 
-**If there is an `Scope`, it will act as the parent to any newly started `Span`** unless the programmer invokes `ignoreActiveSpan()` at `buildSpan()` time, like so:
+**If there is an `Scope`, it will act as the parent to any newly started `Span`** unless 
+the programmer invokes `ignoreActiveSpan()` at `buildSpan()` time or specified parent context explicitly:
```

Comment:
Not directly related to this PR, but wondering whether `ignoreActiveSpan` should be changed to `ignoreScope` - to avoid confusion later?

Reply:
could be :)